.github/workflows/gradle.yml

```diff
@@ -3,63 +3,77 @@ name: Java CI
 on: 
   push: 
     branches: 
-      - main
+      - '*'
```

Comment:
If we are changing the CI to run on all branches, then we can just have one? ```push or pull_request``` The build is running twice now.

Reply:
Changed to only run on pull requests.